### ISSUE TITLE
Data registra partnerov verejneho sektora

### ISSUE THREAD

**Issue by jsuchal** [slovensko-digital/harvester.ecosystem#5]
Toto by sa hodilo mat u nas. 

https://rpvs.gov.sk/opendata/swagger/ui/index

https://rpvs.gov.sk/OpenData/$metadata 

Problematicke vyzera byt na prvy pohlad sledovanie zmien. Lebo tento endpoint https://rpvs.gov.sk/OpenData/Partneri?%24expand=PartneriVerejnehoSektora%2CKonecniUzivateliaVyhod asi nebude usporiadany podla poslednej zmeny.

**Comment by jsuchal** [comment]
Ok kukol som na to vyzera ze by to slo takto:

1. Cez Partneri API budem tahat najnovsize vlozky. Pre kazdu vlozku potom zavolam Partneri/id spolu s expandom na vsetky podentity. https://rpvs.gov.sk/OpenData/Partneri(11192)?%24expand=VerejniFunkcionari%2COpravneneOsoby%2CVymaz%2CKonecniUzivateliaVyhod%2CPartneriVerejnehoSektora

2. Pozriem ci mam tieto subentity u seba. Ak nie tak volam tieto endpointy (cez idcka) na dotiahnutie dalsich info k nim.

- PartneriVerejnehoSektora https://rpvs.gov.sk/OpenData/PartneriVerejnehoSektora?%24expand=Partner%2CPravnaForma%2CAdresa
- OpraveneOsoby https://rpvs.gov.sk/OpenData/OpravneneOsoby?%24expand=Partner%2CAdresa 
- KonecniUzivateliaVyhod https://rpvs.gov.sk/OpenData/KonecniUzivateliaVyhod?%24expand=Partner%2CStatnaPrislusnost%2CPravnaForma%2CAdresa
- https://rpvs.gov.sk/OpenData/VerejniFunkcionari?%24expand=Partner

Vyzera ze osoby nie su nijakym sposobom normalizovane cize tam bude kopec duplicit.

Partneri https://rpvs.gov.sk/OpenData/Partneri?%24expand=Vymaz tuto je skoda ze akonahle sa tam da OpravneneOsoby do `$expand` tak to hadze error. podobne pre VerejniFunkcionari

**Comment by MiroBabic** [comment]
ok pozriem na to

**Comment by vlko** [comment]
Je to take riadne zamaskovane OData v4 ale toto funguje na jeden query:
https://rpvs.gov.sk/OpenData/Partneri?$expand=*,KonecniUzivateliaVyhod($expand=*),PartneriVerejnehoSektora($expand=*),OpravneneOsoby($expand=*),VerejniFunkcionari($expand=*),Vymaz($expand=*),Pokuta($expand=*)

**Comment by MiroBabic** [comment]
tu je verzia z pred par mesiacov, akurat ked si najdem cas dorobim tam tahanie aj statov, podla toho co poradil vlko
https://github.com/MiroBabic/harvester.ecosystem

**Comment by jsuchal** [comment]
@lenhardt nie, toto je trosku problematicky dataset kedze GDPR, cize toto u nas stoji a neplanujeme to davat nateraz do ekosystemu. Kod pokial viem ma @MiroBabic ale  nie je pripraveny na zapojenie do ekosystemu.
